tests/study_tests/test_study.py

```diff
@@ -799,9 +799,18 @@ def _objective(trial: Trial) -> float:
 def test_optimize_with_progbar_parallel(capsys: _pytest.capture.CaptureFixture) -> None:
 
     study = create_study()
-    with pytest.warns(UserWarning, match="Progress bar only supports serial execution"):
-        study.optimize(lambda _: 1.0, n_trials=10, show_progress_bar=True, n_jobs=-1)
```

Comment:
Could you add similar test to this part for `timeout`-based progress bar with `n_jobs!=1`?

Reply:
Ok ,fixed.